net-analyzer/ospd/ospd-1.3.2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
```

Comment:
Does it really only support python3.6? Same comments as in the previous python-related package.

Reply:
https://github.com/greenbone/ospd/blob/ospd-1.3/INSTALL
Prerequisites:
* python 3

Reply:
But remember that Gentoo provides python3.4, python3.5, python3.6 and python3.7 currently. python3.6 is the default and always a safe bat, but IF your program works with other versions, you should enable them for the user. distutils-r1 has DISTUTILS_SINGLE_IMPL variable to use setuptools but install for one implementation only.